packages/framework/src/Form/Admin/Customer/UserFormType.php

```diff
@@ -132,7 +132,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     new Email(['message' => 'Please enter valid e-mail']),
                     new UniqueEmail(['ignoredEmail' => $user !== null ? $user->getEmail() : null]),
                 ],
-                'label' => t('Email'),
+                'label' => t('E-mail'),
```

Comment:
A new translation dump should be made after this change (removing the "Email" message)